src/courses/mappers/02.md

```diff
@@ -0,0 +1,128 @@
+---
+order: 2
+next: 03.md
+title: What Is OHDF?
+author: Charles Hu
+---
+
+## What is OHDF?
+
+OASIS Heimdall Data Format (OHDF) is a security data format used to normalize generated data exports from various security tools into a single common data format usable by the Security Automation Framework (SAF) tool suite. The format is defined by the [OHDF schema](https://saf.mitre.org/framework/normalize/ohdf-schema) and its goal is to provide a simple and intuitive means for representing security validation profiles, controls, and results.
+
+[OASIS Open](https://www.oasis-open.org/) is an international standards body that works on the development and advancement of open source technological standards. OHDF is currently in the process of becoming an OASIS Open standard. For more information on the OHDF charter for OASIS Open, [refer here](https://groups.oasis-open.org/communities/tc-community-home2?CommunityKey=f8888caa-8401-46f8-bf10-018dc7d3f577).
+
+## Why OHDF?
+
+- Many security tools do not provide context to relevant compliance standards for comparison across security tools.​
```

Comment:
change: Many security tools do not provide context to relevant **compliance** standards for comparison across security tools.
to: Many security tools do not provide context to relevant **security** standards for comparison across security tools.

Reply:
Addressed.